File: solutions/julia/raindrops/1/raindrops.jl
function candiv(n,d,str)
    rem(n,d)==0 ? str : ""
end

function raindrops(number)
   res = candiv(number,3,"Pling") * candiv(number,5,"Plang") * candiv(number,7,"Plong")
   isempty(res) ? string(number) : res
end

File: solutions/haskell/raindrops/1/src/Raindrops.hs
module Raindrops (convert) where

convert :: Int -> String
convert n =
  let
    str = (ping 3 "Pling") ++ (ping 5 "Plang") ++ (ping 7 "Plong")
  in
    if null str then show n else str
    where
      ping n2 excl = if mod n n2 == 0 then excl else ""
      